src/use-cases/queries/RecupererUneGouvernance.ts

```diff
@@ -19,16 +20,64 @@ export type ComiteReadModel = Readonly<{
   dateProchainComite: Date
   nom: string
   periodicite: string
+  type : TypeDeComite
 }>
 
 export type FeuilleDeRouteReadModel = Readonly<{
-  budgetGlobal: number
   nom: string
+  porteur: MembreReadModel
   totalActions: number
+  budgetGlobal: number
+  actions: ReadonlyArray<Action>
+  montantSubventionDemande: number
+  montantSubventionAccorde: number
+  montantSubventionFormationDemande: number
+  montantSubventionFormationAccorde: number
+  beneficiaires: ReadonlyArray<MembreReadModel>
```

Comment:
```suggestion
  beneficiairesSubvention: ReadonlyArray<MembreReadModel>
```
Histoire d'être plus explicite